src/hooks/useConnectionForm.ts

```diff
@@ -342,6 +342,46 @@ export const useConnectionForm = (validationSchema: ZodObject<ZodRawShape>, mode
 			Integrations.slack,
 			Integrations.linear,
 		]);
+		const oauthType = migratedAuthIntegrations.has(integrationName as Integrations)
+			? ConnectionAuthType.OauthDefault
+			: ConnectionAuthType.Oauth;
```

Comment:
Only migrated integrations using this function (`handleOauth`), so you don't need to check if they are in the `migratedAuthIntegrations` Set.

Reply:
`oauthType` should always be `ConnectionAuthType.OauthDefault`